playbooks/roles/common/files/get-agent.sh

```diff
@@ -10,7 +10,7 @@ readonly ARCH="$(uname -m)"
 readonly ME="$(basename "$0")"
 readonly HERE=$(CDPATH='' cd "$(dirname "$0")" && pwd -P)
 
-readonly DEFAULT_DOWNLOAD_SITE="https://download-files.appdynamics.com"
+readonly DEFAULT_DOWNLOAD_SITE=""
```

Comment:
we probably do not have to change this script, as it's not going to be used in this flow;

Reply:
In the future, we may control download location with a variable, so that you can choose the downloads method and hopefully merge these changes to the main repo as well.
In case that Artifactory we by accident refer to this script, access would only be denied from their internal network.